themes/contrib/emulsify/components/_patterns/03-organisms/site/site-header/_site-header.scss

```diff
@@ -49,10 +88,31 @@
         }
 
         .menu {
+            @extend %transition;
             display: none;
 
             @include medium {
                 display: block;
+                transition-property: margin;
+            }
+
+            &.is-active {
+                background: rgba(1,49,88,0.8);
```

Comment:
@alexabbott Is this a color that's already in our variables file? If so, you can do something like `rgba($navy, 0.8)`. If not, please add it to the colors file, and use the new variable here.